src/ipc/ipc4/helper.c

```diff
@@ -702,13 +718,58 @@ const struct comp_driver *ipc4_get_drv(uint8_t *uuid)
 	return drv;
 }
 
+#if CONFIG_LIBRARY
+struct ipc4_module_uuid {
+	int module_id;
+	struct sof_uuid uuid;
+};
+
+const struct ipc4_module_uuid uuid_map[] = {
```

Comment:
Can you add a TODO comment that "Hardcoded table of module UUIDs, to be replaced a better mechanism later on."